controller/templates/controller/index.html

```diff
@@ -2,3 +2,121 @@
 {% block title %}
   Controller
 {% endblock title %}
+{% block extra_css %}
+  <style>
```

Comment:
Most of the things copied over don't do anything in this context. I've marked those that do, we can remove the rest.

Reply:
All of this copied over CSS is what I mentioned in [this comment](https://github.com/ImperialCollegeLondon/drunc_ui/pull/188#discussion_r1825664892) on the PR that introduced it. We should probably find a better way to include it.

Reply:
Opened #214.